src/tigerbeetle-java/src/main/java/Benchmark.java

```diff
@@ -26,7 +26,8 @@ public static void main(String[] args) {
                 throw new Exception("Unexpected createAccount results");
 
             final int max_batches = 3;
-            final int max_transfers_per_batch = 8191; // config.message_size_max - @sizeOf(vsr.Header)
+            final int max_transfers_per_batch = 8191; // config.message_size_max -
```

Comment:
I know this is only benchmark but why hardcode this and not get the size via the C FFI?

Reply:
The C client does not expose neither the `config.message_size_max` nor the `sizeOf(vsr.Header)`.

But I agree, it shouldn't be a magic number like that.